### DISCUSSION TITLE
openssl build for High Sierra?

### DISCUSSION THREAD

**Discussion by michaelsbradleyjr** [Homebrew/discussions#1264] [Everyday usage] [answered]
In view of bintray shutting down, I was exploring what's available e.g. here: https://github.com/orgs/Homebrew/packages/container/package/core%2Fopenssl%2F1.1.

This is the bintray URL I was using: https://bintray.com/homebrew/bottles/download_file?file_path=openssl%401.1-1.1.1g.high_sierra.bottle.tar.gz.

I don't see a manifest for High Sierra in the GitHub packages page. Is that intentional?

Also, I see the instructions at the top of the packages page explaining how to use docker to get the package, but is it possible to retrieve a build directly from a URL as one could do with a bintray URL?

**Comment by SMillerDev** [comment, marked as answer]
> I don't see a manifest for High Sierra in the GitHub packages page. Is that intentional?

High Sierra is no longer supported so yes.

> but is it possible to retrieve a build directly from a URL as one could do with a bintray URL?

Yes, this is what brew does.

**Reply by michaelsbradleyjr** [reply]
For any interested in knowing exactly how to download a bottle from GitHub Packages, this is what I came up with:

```
curl -L -o openssl.tar.gz -u _:_ $(brew info --json=v1 openssl | jq '.[0].bottle.stable.files.mojave.url' | sed -e 's/^"//' -e 's/"$//')
```

**Reply by carlocab** [reply]
You don't need `sed` here; just pass `-r` to `jq`.
